build-system/tasks/extension-generator/insert-extension-bundles-config.js

```diff
@@ -58,7 +58,7 @@ function insertExtensionBundlesConfig(bundle) {
     })
   );
 
-  execOrThrow(`npx prettier --write ${extensionBundlesJson}`);
+  execOrThrow(`npx prettier --write ${extensionBundlesJson}`, '');
```

Comment:
Same as before, make this an explicit message like `Could not format extension bundle`.